module/VuFind/tests/unit-tests/src/VuFindTest/RecordDriver/EDSTest.php

```diff
@@ -324,7 +526,41 @@ public function testHasPdfAvailable(): void
      */
     public function testGetPdfLink(): void
     {
-        $this->assertEquals(false, $this->getDriver()->getPdfLink());
+        $driver = $this->getDriver('valid-eds-record');
+        $this->assertEquals('pdf ebook url test', $driver->getPdfLink());
+    }
+
+    /**
+     * Test getEbookLink for a record.
+     *
+     * @return void
+     */
+    public function testGetEbookLink(): void
```

Comment:
Should the test method name reflect the fact that we're not expecting to find matching data here?